ultralytics/engine/trainer.py

```diff
@@ -239,16 +239,16 @@ def calc_weights(self):
             print("Not using class weights")
         elif isinstance(cls_weights, list) and len(cls_weights) == nc:
             cls_weights = torch.Tensor(cls_weights)
-        elif cls_weights == "median":
-            cls_weights = self.get_median_frequency_weights(nc)
+        elif cls_weights == "median-inverse":
+            cls_weights = self.get_inverse_class_frequency_weights(nc, median_norm=True)
         elif cls_weights == "inverse": 
             cls_weights = self.get_inverse_class_frequency_weights(nc)
         else:
             raise ValueError("Invalid value for cls_weights. Please use None, a list of length num_classes, 'median', or 'inverse'.")
```

Comment:
Please update the options given by the error message. 

ie 'inverse' to 'median-inverse' if that's the option we end up going with

Reply:
done, lots of new options now, check it out